clippy_dev/src/main.rs

```diff
@@ -164,6 +164,8 @@ fn print_lints() {
 fn update_lints(update_mode: UpdateMode) {
     let lint_list: Vec<Lint> = gather_all().collect();
 
+    let internal_lints = Lint::internal_lints(lint_list.clone().into_iter());
+
     let usable_lints: Vec<Lint> = Lint::usable_lints(lint_list.clone().into_iter()).collect();
     let lint_count = usable_lints.len();
```

Comment:
Might be worth renaming this to `usable_lint_count` to make clear it's only supposed to count usable lints.